### PR TITLE
add links and info for team 293

### PR DESCRIPTION
**Pull request by PeterMitrano**
Repp'n the home team plus shameless self-promotion.

### CODE REVIEW

File: _frc/200/293.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ team:
     - SolidWorks
     - Hopewell Valley Central High School
   links:
-    Website: 
+    Website: http://spike293.weebly.com/
+    Github: http://github.com/Team293
+    The Blue Alliance: http://www.thebluealliance.com/team/293
```
Comment (virtuald) [inline _frc/200/293.md:20]: TBA is automatically linked for all teams, not necessary.

**Comment by virtuald** [comment]
Also, there's a page on your team in the original FIRSTwiki, do you want to migrate any of that or is it completely out of date (though, I've been keeping out 'accomplishments', as that information can be found at TBA)

https://github.com/firstwiki/original_archive/blob/master/index.php/293/index.html.md

**Comment by PeterMitrano** [comment]
Oh sweet... I never knew about the old one. I'll transfer some of the info

**Comment by PeterMitrano** [comment]
Am I expected to check this by running it locally or nah.

**Comment by virtuald** [comment]
If you made complex changes... sure, but otherwise it doesn't matter as long as the content reads well. The Travis-CI check will fail if you mess anything up. :)

**Pull request merged**